frontend/src/components/common/form/AutoCompleteText.tsx

```diff
@@ -17,6 +17,7 @@ export type IAutoCompleteProps = {
   required?: boolean;
   /** to determine whether to show the code or label defaults to label*/
   showAbbreviation?: boolean;
+  getValueDisplay?: () => string;
```

Comment:
I see you added this new prop to the auto-complete but nothing changed in **the actual implementation** - could you please share more about this prop? what does it do?